mmv1/products/datafusion/api.yaml

```diff
@@ -262,3 +262,32 @@ objects:
               The resource name of the Pub/Sub topic. Format: projects/{projectId}/topics/{topic_id}
             required: true
             input: true            
+      - !ruby/object:Api::Type::Array
+        name: 'accelerators'
+        description: |
+          List of accelerators enabled for this CDF instance.
+        item_type: !ruby/object:Api::Type::NestedObject
+          name: 'accelerator'
+          description: 'Identifies Data Fusion accelerators for an instance.'
```

Comment:
We probably don't need this two lines here and we can see that it's not generated in the downstream https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-6851-old..auto-pr-6851